resources/puppetlabs/lein-ezbake/template/foss/ext/debian/ezbake.init.erb

```diff
@@ -175,18 +182,19 @@ case "$1" in
         get_status_q || exit 0
         do_restart
         ;;
-    restart|force-reload)
-        #
-        # If the "reload" option is implemented then remove the
-        # 'force-reload' alias
-        #
+    restart)
         [ "$VERBOSE" != no ] && log_daemon_msg "Restarting $DESC" "$NAME"
         do_restart
         ;;
+    force-reload|reload)
+        [ "$VERBOSE" != no ] && log_daemon_msg "Reloading $DESC" "$NAME"
+        do_reload
+        log_end_msg $?
```

Comment:
@haus I couldn't find any documentation on it so I wrote a silly script calling `log_end_msg` 

` do_stuff() {
    return 0
}

do_more() {
    return 3
}

do_stuff
log_end_msg $?`

do_more
log_end_msg $?

and the output is:
`...done.`
`...fail!`

It doesn't seem like it exits the script. Just logs whether or not the return is 0. It also does not return any value, so I'll capture the return value of `do_reload`